pom.xml

```diff
@@ -216,6 +217,62 @@
     </build>
 
     <profiles>
+		<profile>
+			<id>rpm</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>rpm-maven-plugin</artifactId>
+						<version>2.1.5</version>
+						<executions>
+							<execution>
+								<id>generate-rpm</id>
+								<goals>
+									<goal>rpm</goal>
+								</goals>
+							</execution>
+						</executions>
+						<configuration>
```

Comment:
picks up the description and version info from the project, forgot to bring the license when moving to the profile. This is why I can't have nice things Ewen. 

rpm -qip kafka

Name        : kafka-connect-jdbc           Relocations: (not relocatable)
Version     : 3.3.0                             Vendor: Confluent, Inc.
Release     : SNAPSHOT20170201063236        Build Date: Wed Feb  1 01:32:56 2017
Install Date: (not installed)            Build Host: ryans-mbp.attlocal.net
Group       : Applications/Internet         Source RPM: kafka-connect-jdbc-3.3.0-SNAPSHOT20170201063236.src.rpm
Size        : 6413778                          License: Apache License 2.0
Signature   : RSA/SHA1, Wed Feb  1 01:32:56 2017, Key ID e64737809b06408d
Packager    : Confluent Packaging
URL         : http://confluent.io
Summary     : kafka-connect-jdbc
Architecture: noarch
Description :
A Kafka Connect JDBC connector for copying data between databases and Kafka.